dre-service/src/main/java/io/cdap/re/RulesDB.java

```diff
@@ -325,14 +323,20 @@ void createRulebook(NamespacedId namespacedId, Rulebook rulebook)
     List<String> ruleIds = new ArrayList<>();
 
     for (Rule rule : rulebook.getRules()) {
-      RuleRequest request = new RuleRequest(rule.getName(), rule.getDescription(), rule.getWhen(), rule.getThen());
-      ruleIds.add(rule.getName());
+      // Have to remove the parenthesis introduced when importing a rulebook
+      String when = rule.getWhen();
+      when = when.replaceAll("\\(", "");
```

Comment:
I'm confused about why this is needed. Did something change during the refactoring? Why would import add things that shouldn't be there?

Reply:
I'm not sure if something changed during the refactoring; I tried my best not too. However, I was having issues when exporting and importing rulebooks. For example, the following is part of an exported rulebook.

```
rule test {
    description 'asdf'
    when(asdf) then {
      asdf;asdffffff;
    }
  }
```

When the rulebook was imported, it created a rule if it was not already present. The rule that was created had the when `(asdf)` instead of `asdf` (i.e., without the parenthesis). I thought that was not the intended behavior since the example of the README did not have parenthesis in the when. 

Also the thens had an extra empty space at the end of each then. So, I trimmed the then strings when creating a rule.

Reply:
Can you check the behavior of the current develop branch? The namespace refactoring shouldn't change import/export behavior at all